fplll/bkz_param.h

```diff
@@ -91,6 +91,7 @@ class BKZParam
           - BKZ_GH_BND        use the Gaussian heuristic to reduce the enumeration bound of possible
           - BKZ_SD_VARIANT    run SD-BKZ
           - BKZ_SLD_RED       run slide reduction
+          - BKZ_PRUNE_ONLY    only run pruner
```

Comment:
Same as above, I don't see how this is a `BKZ_` thing.

Reply:
Fixed.